.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: trusty
 
 env:
   global:
-    - CONDA_DEPS="pip pytest numpy pillow pandas requests scipy nltk six tqdm seaborn matplotlib pathos tensorflow"
+    - CONDA_DEPS="pip pytest numpy pillow pandas requests scipy nltk six tqdm seaborn matplotlib pathos tensorflow contextlib2"
```

Comment:
Added this package to do nested, looping context managers in the Clarifai extractor (see below)